tsl/src/bgw_policy/compression_api.c

```diff
@@ -42,6 +42,7 @@
 #define CONFIG_KEY_COMPRESS_AFTER "compress_after"
 #define CONFIG_KEY_RECOMPRESS_AFTER "recompress_after"
 #define CONFIG_KEY_RECOMPRESS "recompress"
+#define CONFIG_KEY_MAXCHUNKS_TO_COMPRESS "maxchunks_to_compress"
```

Comment:
It's not very clear to me what `CONFIG_KEY_MAXCHUNKS_TO_COMPRESS` means when it is `0`. In the code, `0` means unlimited.

Reply:
Yes, that would be an invalid value. We expect >= 1. Will be documented.